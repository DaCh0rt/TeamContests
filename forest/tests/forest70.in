301 201
-6564 3563
4608 -2046
2722 9093
1060 1118
-1702 5153
-4395 -7363
-4596 -7773
9673 -9574
-5000 -7243
-1058 2421
1822 7852
-2923 -4622
4656 -8634
-4624 -7947
4030 1509
-1819 -4442
-2176 6565
-7742 -2275
-7595 -4579
-689 758
7206 4035
-2407 7852
-4492 -9413
-3014 -9677
2097 -8568
1227 -2344
4131 -2782
8165 6204
772 -3955
-3785 2999
-8968 -2053
4627 1553
1228 -8114
-7901 6080
-9953 4100
-2074 3638
-877 1389
-6746 -7032
9795 -303
-8047 7657
-9894 -111
8686 4434
-7054 -5764
-9219 -4172
-6570 1894
-5913 -8028
-260 8630
5203 -4124
-5396 -8531
-7936 -3341
8185 6533
-5046 5254
-1219 -3375
2406 1758
1095 3852
-727 7275
7362 3283
8782 1500
8932 -4823
5483 8763
9601 -9465
-9307 -2027
-4481 -1007
7395 -9512
-3860 -9617
-3003 2991
-2242 1665
1883 -4405
3563 -842
7819 -425
-4830 -4524
8867 -6372
-8676 3854
4631 3544
-343 -6334
7707 7172
-6759 -2945
5087 5923
7605 -5570
-6400 7910
6251 -6546
9887 -102
-2010 7253
5798 6621
205 3369
6291 -5977
356 -1627
-5621 -4442
-5796 782
-9651 -9388
1817 3150
-6257 -8595
-1597 -8809
-7076 360
-6764 5196
3065 -2408
-1556 -6595
-3982 -5897
-8890 -4302
-6387 -7686
-5826 -8065
3687 5579
5274 -9117
3399 -4617
-6025 6044
-3547 5272
-6314 7799
6739 -2027
7369 -9065
-3829 -8167
6862 2166
-8599 993
-3704 -1217
4569 2413
-85 3554
-7923 -2179
7388 6360
6767 -4403
3277 8898
-5883 -3032
-5591 -1051
-14 3329
-4897 9294
8465 -4823
4867 -2726
-7639 -7659
7793 -9947
9529 -8177
-2512 -1176
-7295 788
-4369 -2850
9480 1715
-7719 -2871
-6944 1990
7916 -5733
4310 -3363
-9590 838
-8306 -736
9579 7786
1082 -7273
-8087 -8996
-2590 2161
-1612 -6780
-3601 -4519
2168 -2969
-1953 663
4342 5822
-4586 2535
4604 -2706
-4867 -1222
-5336 6952
-2004 2252
-5942 -651
9679 -3252
2011 7536
6823 -7857
-820 9658
3880 5173
-4419 -6003
-1728 5950
3749 7872
-7720 -6013
-8180 9548
7372 -516
5647 9177
-2650 -1037
7389 9593
6187 -4729
1294 3212
-782 8623
-7830 9388
-6752 -4295
-7962 2659
-7145 -5972
-5558 4631
1008 -2703
-633 -3237
3837 566
-137 6777
-367 -356
-213 5505
2437 -4906
-1517 -8505
9725 -8824
-6506 -4823
6138 -5709
4389 2767
1004 -5606
-1986 9001
-5407 -2387
-6368 2443
4107 3386
-2094 -8178
-4007 1609
124 -8563
370 8249
-4797 1876
-3809 8506
5185 8738
-3060 -7299
9156 2540
-2043 -4081
-3090 7827
1624 -4085
-5265 -1796
6394 9104
-1475 7255
6046 -3805
-2369 -5706
-6015 -3861
-4560 -3545
7141 5956
-9505 -3780
-9637 7514
-3946 -8031
-1052 1709
-9805 8019
-6878 -7776
-8153 -3329
-2679 3107
-5316 5643
-7334 6981
8818 -2805
-2616 -3231
3327 6060
1790 7010
-4353 109
-7045 3317
6642 7160
-3409 8131
-3544 758
143 7323
1993 -5804
-9732 -4522
-8966 -5030
9728 6912
-1201 -1659
5237 4389
133 6578
-9797 -4404
-1236 7974
-1488 -7136
2831 5510
1008 -765
-4291 3160
9991 -9403
7642 -2205
3256 -2937
-9137 -7103
-9700 -6196
-4050 3857
7426 8696
-1314 -3955
-1608 -9331
6771 8572
-7578 1277
7255 4101
7071 3581
-1552 1767
-2197 4931
3678 -1947
-4526 -9021
-5575 -6874
1045 5682
-5987 9516
-2060 -9895
149 6037
3521 -3207
-7699 3910
-8637 -8981
-6101 5898
-7903 -4601
-4934 9899
216 793
-8632 7655
-6288 -8479
434 4499
4031 -510
-7653 574
1638 2489
2242 735
1924 -200
423 -5492
8603 -5300
-5066 -6724
-6175 -8838
1869 6182
-2576 2715
1981 9557
-2226 490
-4227 -7595
4306 -2469
9433 -630
9679 6194
-214 -7084
2997 2650
-277 2070
6841 4837
-156 1477
3723 -441
-9224 -9393
