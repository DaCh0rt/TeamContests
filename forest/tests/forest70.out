76281.33 395625637.80
